LICENSE-README.md

```diff
@@ -0,0 +1,22 @@
+Copyright (c) 2023 See the [README.md](README.md#contributors) for authors/contributors (bottom of the document).
+
+This software is a digital implementation of the "FüSim MANV"/"FueSim MANV" (Führungssimulation Massenanfall von Verletzen), a German paper-based simulation game for training emergency medical services leadership personnel in managing Mass Casualty Incidents.
+
+The software is licensed with the [GNU Affero General Public License](LICENSE.md) as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+Exceptions apply to image files in the frontend/src/assets folder, which may be covered by and therefore distributed under different licenses.
+The individual license for each image can be found in [frontend/src/assets/LICENSES-IMAGES.md](frontend/src/assets/LICENSES-IMAGES.md). Files added to this project may only use this exception if these files serve the only purpose of being displayed and will only be parsed for the purpose of displaying them.
+Another exception to the default licence exists for datasets. Datasets are files that include no new features and just contain different compositions allowed through template classes given in the source code. Datasets must always be in human readable formats (for example csv, JSON, txt). Datasets license must be mentioned in a LICENSE-DATASET.md (or LICENSES-DATASETS.md) file in the respective folder where the dataset is placed. All licence files must state explicitly which file (or files) they refer to, who the authors are and which license applies (or licenses apply). Datasets are not allowed to create any new features or enhance the software in any way, as to create elements that can be part of an export and are importable into the distributed source code (this software) under the [GNU Affero General Public License](LICENSE.md) as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
```

Comment:
Is there any particular reason datatsets must be human readable? This would cause trouble if someone wants to support compressed exports. It is quite likely this may be desired considering the current size of exercise exports.

Reply:
> Is there any particular reason datatsets must be human readable? This would cause trouble if someone wants to support compressed exports. It is quite likely this may be desired considering the current size of exercise exports.

The text doesn't state that exports must be human readable, but that datasets themselves need to be.
What could be a problem is that `importable into the distributed source code (this software)`, this could be made more clear that with this the running software resulting of the source code is meant (=this program or so).
But we could also add that exports can be compressed via this software and decompressed by import or so.

Do you think about compression standards as zip/tar or so or do you mean the export being a binary with our own datastructure for more efficient file sizes, not the current `json`?

Reply:
Actually the dataset definition could include exports / imports, I commited a change that should make compressioned and decompressioned exports and imports possible.
@Nils1729 are you okay with this change (https://github.com/hpi-sam/digital-fuesim-manv/pull/636/commits/08916c6cd309cce308c6706f62868feb24c11f4e)?